File: app/src/main/java/com/anokmik/tripassistant/databinding/components/ImageViewComponent.java
package com.anokmik.tripassistant.databinding.components;

import android.databinding.BindingAdapter;
import android.widget.ImageView;

import com.squareup.picasso.Picasso;

public final class ImageViewComponent {

    @BindingAdapter("android:src")
    public void loadImage(ImageView view, String imageUrl) {
        Picasso.with(view.getContext()).load(imageUrl).into(view);
    }

}

File: app/src/main/kotlin/com/anokmik/tripassistant/databinding/components/ImageViewComponent.kt
package com.anokmik.tripassistant.databinding.components

import android.databinding.BindingAdapter
import android.widget.ImageView

import com.squareup.picasso.Picasso

class ImageViewComponent {

    @BindingAdapter("android:src")
    fun loadImage(view: ImageView, imageUrl: String) {
        Picasso.with(view.context).load(imageUrl).into(view)
    }

}